src/api_clients/sqs.py

```diff
@@ -0,0 +1,117 @@
+import json
+import time
+import uuid
+from dataclasses import dataclass
+from enum import Enum
+from typing import Optional, List
+from typing import Union
+
+import boto3
+import prefect
+from prefect import task
+from pydantic import BaseModel, root_validator
+
+from utils.config import SQS_MESSAGE_VERSION, SQS_REC_QUEUE, SQS_PROSPECT_QUEUE
+
+ONE_DAY_S = 86400
+ONE_WEEK_S = ONE_DAY_S * 7
+
+
+# todo: remove sqs when we finished migration to SageMaker Feature Store
+
+class RecommendationCandidate(BaseModel):
+    item_id: int
+    publisher: str
+    feed_id: Optional[int] = None
+
+
+class CurationProspect(BaseModel):
+    scheduled_surface_guid: str
+    prospect_id: str
+    url: str
+    prospect_source: str
+    save_count: int
+    predicted_topic: str
+    rank: int
+
```

Comment:
@eu9ene in case the context is missing, we attempted to reuse the SQS queue and code for two distinct purposes in the metaflow repo.  The first was generating the recommendation candidate sets for use in the recommendation-api.  The second was generating curation prospect candidate sets that are ingested by the curation tool (and editorial team).  that is how the two models above came to be.  we now have separate SQS queues as well